RELEASE_NOTES.md

```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
 
 This debugging feature will mostly be useful to people who are already Ruby experts.
 
+### New ksh resource
 
+ksh is now available to use as a resource and is available as an interpreter in the script resource.
```

Comment:
I might lead with what ksh is, e.g. "Korn Shell scripts can now be run using the `ksh` resource or by setting the interpreter parameter of the script resource to ksh."